2020-09-21 12:43:10,181 - Adb_Tools.py [line:93] - ERROR: 请检查目标设备是否与主机连接成功！！！
2020-09-21 12:43:10,181 - Adb_Tools.py [line:62] - ERROR: local variable 'i' referenced before assignment
2020-09-21 12:44:53,732 - Adb_Tools.py [line:93] - ERROR: 请检查目标设备是否与主机连接成功！！！
2020-09-21 12:44:53,732 - Adb_Tools.py [line:62] - ERROR: local variable 'i' referenced before assignment
2020-09-21 12:45:12,058 - Adb_Tools.py [line:93] - ERROR: 请检查目标设备是否与主机连接成功！！！
2020-09-21 12:45:12,058 - Adb_Tools.py [line:62] - ERROR: local variable 'i' referenced before assignment
